webapp/src/main/webapp/src/app/app.routes.ts

```diff
@@ -19,35 +20,56 @@ export const routes: Routes = [
         path: '',
         pathMatch: 'prefix',
         redirectTo: 'groups'
-      }, {
+      },
+      {
         path: 'groups',
         pathMatch: 'prefix',
         data: { breadcrumb: { translate: 'labels.groups.title' } },
```

Comment:
Interesting thing I found with Will was that the breadcrumb defs need to be on the route entries with paths. Not the ```page: ''``` children.